--- rarpd-1.1/configure~	1999-11-17 00:51:51.000000000 +0000
+++ rarpd-1.1/configure	2003-07-03 21:11:26.000000000 +0000
@@ -1687,7 +1687,7 @@
 echo "configure:1688: checking for libnet config in $i" >&5
       
 ac_cv_found_libnet_bin=no
-if test -f "$i/libnet-config" ; then
+if test -f "$i/libnet1-config" ; then
   ac_cv_found_libnet_cfg=yes
 fi
 
@@ -1713,7 +1713,7 @@
       
 
 saved_LIBS=$LIBS
-LIBS="$saved_LIBS -L$i -lnet"
+LIBS="$saved_LIBS -L$i -lnet1"
 cat > conftest.$ac_ext <<EOF
 #line 1719 "configure"
 #include "confdefs.h"
@@ -1760,7 +1760,7 @@
 
 
 if test "$ac_cv_found_net_lib" = "yes"; then
-  if test \! -f "$i/libnet.a" -a \! -f "$i/libnet.so" -a \! -f "$i/libnet.sl"; then
+  if test \! -f "$i/libnet1.a" -a \! -f "$i/libnet1.so" -a \! -f "$i/libnet.sl"; then
     ac_cv_found_net_lib=no
   fi
 fi
@@ -1786,7 +1786,7 @@
 echo "configure:1787: checking for libnet header in $i" >&5
       
 ac_cv_found_libnet_inc=no
-if test -f "$i/libnet.h" ; then
+if test -f "$i/libnet1.h" ; then
   ac_cv_found_libnet_inc=yes
 fi
 
@@ -1809,7 +1809,7 @@
 	else
 	  ac_cv_found_libnet=yes
 	  echo "$ac_t""yes" 1>&6
-	  LIBNET_CONFIG=$ac_cv_libnet_where_cfg/libnet-config
+	  LIBNET_CONFIG=$ac_cv_libnet_where_cfg/libnet1-config
 	  LIBNET_INC_DIR=$ac_cv_libnet_where_inc
 	  LIBNET_LIB_DIR=$ac_cv_libnet_where_lib
 
--- rarpd-1.1/rarpd.c~	1999-11-17 00:51:22.000000000 +0000
+++ rarpd-1.1/rarpd.c	2003-07-03 21:14:17.000000000 +0000
@@ -31,7 +31,7 @@
 #include <net/if_ether.h>
 #endif
 #include <pcap.h>
-#include <libnet.h>
+#include <libnet1.h>
 #ifdef HAVE_CTYPE_H
 #include <ctype.h>
 #endif
